charts/sscs-tribunals-api/values.preview.template.yaml

```diff
@@ -148,7 +149,7 @@ xui-webapp:
       SERVICES_IDAM_API_URL: https://idam-api.aat.platform.hmcts.net
       SERVICES_S2S: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
       SERVICES_HEARINGS_COMPONENT_API: http://jurisdiction-hearings-api-aat.service.core-compute-aat.internal
-      SERVICES_HEARINGS_COMPONENT_API_SSCS: http://${SERVICE_NAME}-sscs-hearings-api
+      SERVICES_HEARINGS_COMPONENT_API_SSCS: http://${SERVICE_NAME}-java
```

Comment:
Do we need -java?

Reply:
yes it will point as xui will point to trib pod e.g sscs-tribunals-api-pr-4056-java-bd696cd95-xh66g

Reply:
i think we don't need -java at the end. On similar lines to https://github.com/hmcts/sscs-tribunals-case-api/blob/64584465d933efe208224ff27019cb46a298cc6e/charts/sscs-tribunals-api/values.preview.template.yaml#L31